src/resource-listers/AllResourcesLister.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var _ = require('underscore'),
+    Q = require('q'),
+    AWS = require('aws-sdk'),
+    DynamoResourceLister = require('./base/DynamoResourceLister'),
+    dynamo = new AWS.DynamoDB();
+
+module.exports = DynamoResourceLister.extend({
+
+   _fetchTableNames: function() {
+      return this._fetchAll(dynamo, 'listTables', {}, 'ExclusiveStartTableName', 'LastEvaluatedTableName')
+         .then(function(resps) {
+            return _.flatten(_.pluck(resps, 'TableNames'));
+         });
+   },
+
+   _fetchAll: function(service, methodName, request, requestPaginationTokenName, responsePaginationTokenName) {
+      var deferred = Q.defer(),
+          responses = [],
+          next = null;
+
+      function loop() {
+         var params;
+
+         params = _.extend({}, request);
+         params[requestPaginationTokenName] = next;
+
+         console.log('Calling "%s" (%s: %s)', methodName, requestPaginationTokenName, next);
```

Comment:
Let's remove this logging statement. Up to now the only log statements we've left in the code are directly related to the status of or changes to resources. We don't want to start flooding the logs with debugging info.

At some point we'll resolve this by having a proper logging library so that people can control logging output, which most of our libs are lacking currently.